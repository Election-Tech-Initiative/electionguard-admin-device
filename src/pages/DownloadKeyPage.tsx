import React from 'react'
import Main, { MainChild } from '../components/Main'
import Prose from '../components/Prose'

const DownloadKeyPage = () => {
  return (
    <Main>
      <MainChild center>
        <Prose textCenter>
          <h1>Download Key</h1>
        </Prose>
      </MainChild>
    </Main>
  )
}

export default DownloadKeyPage
