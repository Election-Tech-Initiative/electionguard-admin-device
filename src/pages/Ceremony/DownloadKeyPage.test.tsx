import React from 'react'
import { Route } from 'react-router-dom'

import { render } from '../../../test/testUtils'

import DownloadKeyPage from './DownloadKeyPage'

it('renders DownloadKeyPage', async () => {
  const { container } = render(<Route path="/" component={DownloadKeyPage} />, {
    route: '/',
  })
  expect(container.firstChild).toMatchSnapshot()
})
