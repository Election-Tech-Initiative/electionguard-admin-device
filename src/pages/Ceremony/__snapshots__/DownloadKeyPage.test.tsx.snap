// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders DownloadKeyPage 1`] = `
<main
  class="sc-bdVaJa iiFDUH"
>
  <div
    class="sc-bwzfXH jAEsws"
  >
    <div
      class="sc-htpNat chxRAJ"
    >
      <h1>
        Download Key
      </h1>
    </div>
  </div>
</main>
`;
